src/fdk_organization_bff/config/config.py

```diff
@@ -12,8 +12,8 @@ class Config:
     _ROUTES = {
         "PING": "/ping",
         "READY": "/ready",
-        "ORG_CATALOG": "/organizationcatalog/{id}",
-        "ORG_CATALOGS": "/organizationcatalog",
+        "ORG_CATALOG": "/organizationcatalogs/{id}",
+        "ORG_CATALOGS": "/organizationcatalogs",
```

Comment:
Eller kan du gjøre slik så, du vil vel alltid at 'id' bruker samme path som den for alle:

"ORG_CATALOGS": "/organizationcatalogs",
"ORG_CATALOG": f"{ORG_CATALOGS}/{id}",

Reply:
Litt vanskelig med f-string når vi bruker `{id}`, men kan jo bygge det på et annet vis